pages/docs/guides/validate.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Validate AsyncAPI Documents"
+description: In this guide, we'll learn multiple ways to validate AsyncAPI documents.
+weight: 120
+---
+
+# Introduction
+In this guide, we'll learn multiple ways to validate AsyncAPI documents. 
+
+When people talk about AsyncAPI validation, sometimes they mean completely different things: **validation of AsyncAPI documents** or **validation of messages in runtime** _(message schemas provided in the AsyncAPI document)_.  
+
+# Prequisites
+Because this guide covers multiple options for validating AsyncAPI documents, there are different tool options you might choose to install:
+- [AsyncAPI CLI](https://github.com/asyncapi/cli#installation)
+- [Spectral](https://meta.stoplight.io/docs/spectral/b8391e051b7d8-installation)
+
+# Create AsyncAPI documents
+An AsyncAPI document is a machine-readable definition of your Event-Driven API that can be used to generate documentation and code. You can create an AsyncAPI document with the [AsyncAPI CLI](https://github.com/asyncapi/cli#installation).
+
+Generate a sample `asyncapi.yaml` file (AsyncAPI document) with the following CLI command: 
+
+```
+asyncapi new --example=default-example.yaml --no-tty
+```
+
+# Validate AsyncAPI documents
+Validating an AsyncAPI document requires two different pieces:
+1. Validation against the specification.
+2. Validation against the best practices or company governance rules.
+
+## Validate against specification
+Let's discuss options for validating against the specification. (This ensures that every content of the document is written in accordance with the AsyncAPI specification.) 
+
+There are several tool options you may select for validating against the specification: _AsyncAPI Studio_, _AsyncAPI CLI_, and _Parsers_.
+
+### AsyncAPI Studio validation 
+[AsyncAPI Studio](https://github.com/asyncapi/studio#readme) provides a visual and easy way to validate your AsyncAPI documents against the specification. 
+
+(It uses the [AsyncAPI JavaScript parser](https://github.com/asyncapi/parser-js) behind the scenes to perform syntax checks and validate documents.)
+
+Errors in your document are highlighted with a red underline, showing which lines are invalid. The console also provides feedback, allowing you to further troubleshoot with detailed error messages.
+
+When a document is invalid, it provides the following error: `Empty or invalid document please fix errors / define AsyncAPI document`.
+
+### AsyncAPI CLI validation 
+The following [AsyncAPI CLI](https://github.com/asyncapi/cli#installation) command validates AsyncAPI documents in your local computer or in automation:
```

Comment:
```suggestion
The following [AsyncAPI CLI](https://github.com/asyncapi/cli#installation) command validates AsyncAPI documents in your local computer or in CI/CD automation:
```

Reply:
btw, you can also write that like in case of `Studio` it uses `AsyncAPI JavaScript parser` behind the scenes.

also was wondering, maybe add a tip, that people can also open Studio from CLI 😄 `asyncapi start studio`. This could be a note at the end of section